completion/available/aliases.completion.bash

```diff
@@ -87,9 +91,9 @@ function _bash-it-component-completion-callback-on-init-aliases() {
                         if [[ \$COMP_LINE == \"\$prec_word \$compl_word\" ]]; then
                             prec_word='$alias_cmd $alias_args'
                             prec_word=\${prec_word#* }
-                        fi
+                        fi:
```

Comment:
sorry, but this seems like a typo. I don't recognise this syntax nor could I make it work in a local test...